src/example/counter/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../store/reducer";
+import {
+  increamentCouter,
+  decreamentCouter,
+} from "../../store/modules/example/action";
+
+const Example = () => {
+  const dispatch = useDispatch();
+  const count = useSelector(({ exampleReducer }: RootState) => exampleReducer);
+
+  return (
+    <div>
+      <button
+        onClick={() => {
+          dispatch(increamentCouter());
+        }}
```

Comment:
create a method above and pass the reference here

Reply:
resolved it by passing a reference